### PR TITLE
feat: allow arguments to be passed to tasks

### PR DESCRIPTION
**Pull request by joerdav**

### CODE REVIEW

**Comment by joerdav** [comment]
Fixes #29

**Comment by joerdav** [comment]
Closes #23

**Comment by a-h** [comment]
I love it. Really cool.

File: doc/content/task-syntax/inputs.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+description:
+linkTitle: "Inputs"
+menu: { main: { parent: 'task-syntax', weight: 10 } }
+---
+
+## Pass an input to a task
+
+Task definitions are run as shell scripts, therefore you can pass arguments to them as such.
+
+`xc [task-name] [inputs...]`
```
Comment (a-h) [inline doc/content/task-syntax/inputs.md:12]:
```suggestion
## Task inputs

Some tasks will require inputs such as the name of servers, paths to files, or credentials.

`xc` tasks are executed as shell scripts, so they support shell syntax for variables.

`xc [task-name] [inputs...]`
```

File: doc/content/task-syntax/inputs.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+```sh
+$ xc greet Joe
++ echo 'Hello, Joe.'
+Hello, Joe.
+```
+
+## Variadic Syntax
+
+As xc tasks are executed as shell scripts you can also use positional syntax of arguments.
```
Comment (laurabirchIW) [inline doc/content/task-syntax/inputs.md:114]:
```suggestion
As xc tasks are executed as shell scripts you can also use variadic syntax of arguments.
```

**Pull request merged**